### DISCUSSION TITLE
Task Hooks

### DISCUSSION THREAD

**Discussion by FlorianLudwig** [copier-org/copier#831] [Q&A] [answered]
Hi there,

the documentation states that "task hooks" exist, but I cannot find any other mention of what those are.

I would think it is something like the [hooks from cookiecutter](https://cookiecutter.readthedocs.io/en/2.1.1/advanced/hooks.html), allowing the template creator to execute python before and after generation is run.

Is that what is meant with "task hooks"?

**Comment by pawamoy** [comment, marked as answer]
If you're talking about the [comparison page](https://copier.readthedocs.io/en/stable/comparisons/), then Tasks Hooks there refer to [tasks](https://copier.readthedocs.io/en/stable/configuring/#tasks).
